README.md

```diff
@@ -16,7 +16,7 @@ Clisso uses the [SAML][7] standard to authenticate users.
 
 ## Installation
 
-### Using a Pre-Compiled Binary
+### Using A Pre-Compiled Binary
```

Comment:
Actually the "a" should be lower case... sorry :-)
http://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html